docs/topics/inline-functions.md

```diff
@@ -122,9 +124,30 @@ inline fun f(crossinline body: () -> Unit) {
 }
 ```
 
-> `break` and `continue` are not yet available in inlined lambdas, but we are planning to support them, too.
+### Break and continue
+
+> This feature is currently [Experimental](components-stability.md#stability-levels-explained).
+> We're planning to stabilize it in future releases.
+> To opt in, use the `-Xnon-local-break-continue` compiler option.
+> We would appreciate your feedback on it in [YouTrack](http://kotl.in/issue).
```

Comment:
It's not clear to me how much sense it makes to point to the YouTrack new issue page in such kind of sentence. At least, it's slightly unexpected.

Maybe a better option would be to point to the issue itself https://youtrack.jetbrains.com/issue/KT-1436